frontend/src/components/map/Leaflet.jsx

```diff
@@ -1,19 +1,35 @@
 import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
+import PropTypes from "prop-types";
+
+function Leaflet(props) {
+  const { fuelList } = props;
 
-function Leaflet() {
   return (
-    <MapContainer center={[51.505, -0.09]} zoom={13} scrollWheelZoom={false}>
+    <MapContainer
+      center={[49.259037, 4.031781]}
+      zoom={13}
+      scrollWheelZoom={false}
+    >
       <TileLayer
         attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
         url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
       />
-      <Marker position={[51.505, -0.09]}>
+      <Marker position={[49.216, 4.057]}>
         <Popup>
           A pretty CSS3 popup. <br /> Easily customizable.
         </Popup>
       </Marker>
+      {fuelList.map((fuel) => (
+        <Marker key={fuel.fields.id} position={fuel.fields.geom}>
+          <Popup>
+            A pretty CSS3 popup. <br /> Easily customizable.
+          </Popup>
+        </Marker>
+      ))}{" "}
     </MapContainer>
   );
 }
-
+Leaflet.propTypes = {
+  fuelList: PropTypes.arrayOf.isRequired,
```

Comment:
arrayOf ? should be arrayOf(something) like arrayOf(PropTypes.number) for example

in your case :

    arrayOf(PropTypes.shape({
      fields: PropTypes.shape({
        id: PropTypes.number.isRequired,
        geom: PropTypes.arrayOf(PropTypes.numbers).isRequired,
      }).isRequired,
    }))

more or less : check with npm run lint and in the console of your browser